memory_addr/src/addr.rs

```diff
@@ -187,6 +232,14 @@ macro_rules! def_usize_addr {
             }
         }
 
+        impl core::ops::Sub<$name> for $name {
```

Comment:
Do we need a `Sub`? It should have the same function as `offset_from`, which returns `isize`. But it would be strange when the type is `usize`, such as `usize - usize = iszie`. I suggest removing this and using `offset_from` always.